src/current/molt/molt-fetch.md

```diff
@@ -150,7 +127,46 @@ Cockroach Labs **strongly** recommends the following:
 
 ### Secure cloud storage
 
-- When using [cloud storage](#cloud-storage) for your intermediate store, ensure that access control is properly configured. Refer to the [GCP](https://cloud.google.com/storage/docs/access-control) or [AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-iam.html) documentation.
+- When using [cloud storage](#cloud-storage) for your intermediate store, ensure that access control is properly configured. 
+
+	- If you are using [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-iam.html) for [cloud storage](#cloud-storage):
+
+		- Ensure that the environment variable and access tokens are set appropriately in the terminal running `molt fetch`. For example:
+
+			{% include_cached copy-clipboard.html %}
+			~~~ shell
+			export AWS_REGION='us-east-1'
+			export AWS_SECRET_ACCESS_KEY='key'
+			export AWS_ACCESS_KEY_ID='id'
+			~~~
+
+		- Alternatively, set the `--use-implicit-auth` flag to use [implicit authentication]({% link {{ site.current_cloud_version }}/cloud-storage-authentication.md %}).
+
+		- Ensure the S3 bucket is created and accessible to CockroachDB.
```

Comment:
Should this be `accessible by` instead of `accessible to` since IMPORT is the one trying to access the resource?